src/operations/get_more.ts

```diff
@@ -0,0 +1,44 @@
+import type { Document, Long } from '../bson';
+import type { Callback, MongoDBNamespace } from '../utils';
+import type { Server } from '../sdam/server';
+import { Aspect, AbstractOperation, OperationOptions, defineAspects } from './operation';
+import type { ClientSession } from '../sessions';
+
+/**
+ * @public
+ * @typeParam TSchema - Unused schema definition, deprecated usage, only specify `FindOptions` with no generic
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export interface GetMoreOptions<TSchema extends Document = Document> extends OperationOptions {
```

Comment:
Options passed for the command are `batchSize`, `comment`, `maxTimeMS`, and the inherited options from `OperationOptions`.

Reply:
```suggestion
export interface GetMoreOptions extends OperationOptions {
```
and you can drop the eslint exception comment.